CHANGES_NEXT_RELEASE

```diff
@@ -0,0 +1,2 @@
+- [cosmos-gui] [FEATURE] Create a GUI for Cosmos, in charge os administrating the shared storage cluster (#2)
```

Comment:
This has been added in this PR since it was missing in PR https://github.com/telefonicaid/fiware-cosmos/pull/3